packages/neo-one-website/src/pages/404.tsx

```diff
@@ -3,18 +3,24 @@ import '../polyfill';
 
 import { Redirect } from '@reach/router';
 import * as React from 'react';
+import { RouteData } from 'react-static';
 import { Helmet } from '../components';
-import { CoreLayout } from '../layout';
+import { CoreLayout, DocsLoading } from '../layout';
 
 // tslint:disable-next-line export-name no-default-export
 export default () => (
-  <CoreLayout path="404">
-    <Helmet title="404">
-      <link rel="canonical" href={process.env.REACT_STATIC_PUBLIC_PATH} />
-      <meta name="robots" content="noindex" />
-      <meta httpEquiv="content-type" content="text/html; charset=utf-8" />
-      <meta httpEquiv="refresh" content={`0; url=${process.env.REACT_STATIC_PUBLIC_PATH}`} />
-    </Helmet>
-    <Redirect to="/" noThrow={typeof window === 'undefined'} />
-  </CoreLayout>
+  // @ts-ignore
+  <RouteData Loader={DocsLoading}>
```

Comment:
All of these really delay page loading - instead move the `RouteData` wrapper to just around the blog link and make the blog link direct to `/blog` while it's loading.